app.yaml

```diff
@@ -30,6 +30,7 @@ handlers:
     Vary: Accept-Encoding
 - url: /build
   static_dir: build
+  application_readable: true
```

Comment:
If I remember our meeting correctly, the final solution (after all minification and externalization is done) would not involve the application having to read from /build. Is that correct? If so, I think Ben is saying, implement that, and don't do this halfway.